### PR TITLE
One fix per day 9

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by Arsnael** [comment]
tests are :green_apple: (failing on deployment tests)

**Comment by mbaechler** [comment]
test this please

File: mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/mail/InMemoryAnnotationMapper.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ public void insertAnnotation(MailboxId mailboxId, MailboxAnnotation mailboxAnnot
     public void deleteAnnotation(MailboxId mailboxId, MailboxAnnotationKey key) {
         lock.writeLock().lock();
         try {
-            mailboxesAnnotations.remove(mailboxId, key.asString());
+            mailboxesAnnotations.remove((InMemoryId)mailboxId, key.asString());
```
Comment (remk) [inline mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/mail/InMemoryAnnotationMapper.java]: Why this cast ?

File: server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/authentication/JwtFilterTest.java [hunk 2]
```diff
@@ -66,9 +67,23 @@ public void setUp() {
         jwtFilter = new JwtFilter(jwtTokenVerifier);
     }
 
+    @Test
+    public void handleShouldDoNothingOnOption() throws Exception {
```
Comment (remk) [inline server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/authentication/JwtFilterTest.java]: s/Option/Options

File: mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/mail/InMemoryAnnotationMapper.java (hunk 1 above)
Comment (mbaechler) [reply mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/mail/InMemoryAnnotationMapper.java]: We know the exact type so we can cast safely (it's done in others methods).
It prevents from calling `remove(Object, Object)` method when the target datastructure is actually typed.

Comment (blackheaven) [reply mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/mail/InMemoryAnnotationMapper.java]: space

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**